1122.py

```diff
@@ -0,0 +1,17 @@
+from collections import defaultdict
+from typing import List
+
+class Solution:
+    def relativeSortArray(self, arr1: List[int], arr2: List[int]) -> List[int]:
+        only_arr1 = sorted([e for e in arr1 if e not in arr2])
```

Comment:
`arr2` is an array; therefore, it is quite expensive to check if every `e` in `arr1` is not in `arr2`. Each individual search takes O(N) time, and you did it for all N elements in `arr1`.